### PR TITLE
feat: added Haptic Feedback

### PR DESCRIPTION
**Pull request by Aditi-Singh16**
Fixes issue #1051

### CODE REVIEW

**Comment by nicolasburtey** [comment]
> Fixes issue #1051

thank you for making this PR @Aditi-Singh16 

some comments:

- the formatting is off. you can install/see the error with eslint.
- there are also the receive screens, and lnurl screens that would need the haptic feedback

**Comment by nicolasburtey** [comment]
also, we're following https://www.conventionalcommits.org/en/v1.0.0/ for our commit/PR description

**Comment by Aditi-Singh16** [comment]
> > Fixes issue #1051
> 
> thank you for making this PR @Aditi-Singh16
> 
> some comments:
> 
> * the formatting is off. you can install/see the error with eslint.
> * there are also the receive screens, and lnurl screens that would need the haptic feedback

for receive and lnurl screens, at what event should the feedback be triggered?

File: app/screens/receive-bitcoin-screen/receive-btc.tsx [hunk 1]
```diff
@@ -448,6 +449,12 @@ const ReceiveBtc = () => {
     )
   }
 
+  if (state === PaymentRequestState.Paid) {
+    ReactNativeHapticFeedback.trigger("notificationSuccess", {
+      ignoreAndroidSystemSettings: true,
```
Comment (nicolasburtey) [inline app/screens/receive-bitcoin-screen/receive-btc.tsx:454]: why is that necessary?

**Comment by nicolasburtey** [comment]
PR looks good overall. I have one question. 

Also, some notes: we try to avoid merge, instead we rely on rebase when we want to get on par with main. it makes the git history much cleaner. basically you do that by making sure main is up to date, then in you branch you do `git rebase main`. feel free to google if you don't know the concept of `git rebase`.

File: app/screens/receive-bitcoin-screen/receive-btc.tsx (hunk 1 above)
Comment (Aditi-Singh16) [reply app/screens/receive-bitcoin-screen/receive-btc.tsx:454]: this triggers the haptic feedback when the receive screen is on and that is paid( basically when the tick mark icon comes up).

**Comment by Aditi-Singh16** [comment]
Hi, I got some errors while rebasing and was unable to solve, so created a new PR
